ledger/participant-local-store/src/test/lib/com/daml/platform/localstore/UserStoreTests.scala

```diff
@@ -207,9 +241,17 @@ trait UserStoreTests extends UserStoreSpecBase { self: AsyncFreeSpec =>
         for {
           res1 <- tested.createUser(newUser("user1"), Set.empty)
           res2 <- tested.createUser(newUser("user2"), Set.empty)
-          users1 <- tested.listUsers(fromExcl = None, maxResults = 10000)
+          users1 <- tested.listUsers(
```

Comment:
test for updating idp in a user missing below in this file

Reply:
added